packages/ia-components/sandbox/icons/icon.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Archive from './customsvg/ia-logo-white-icon';
+import iconfile from './iconfiles';
+
+function defaultColor(color, name) {
+	const icon = iconfile.icons.find(icon => icon.tags[0] === name || icon.tags[1] === name);
+
+	if (!color || color === '')
+		return icon.defaultColor
+	else return color;
```

Comment:
Can we refactor the `else` return conditions to remove the else keyword and simply `return color` to avoid being overly verbose? Since the truthy condition exits early, the only time the following lines would execute is if we've reached the `else` condition anyway.